netty/src/main/java/io/grpc/transport/netty/NettyClientHandler.java

```diff
@@ -137,15 +137,16 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
   @Override
   public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise) {
     try {
-      if (msg instanceof CreateStreamCommand) {
-        createStream((CreateStreamCommand) msg, promise);
+      if (msg instanceof CreateStreamNettyCommand) {
```

Comment:
Ok, but maybe we should pick better names to indicate direction of the command (inbound vs outbound).  See my previous comment https://github.com/grpc/grpc-java/pull/367/files#r29995940